.circleci/config.yml

```diff
@@ -40,15 +51,35 @@ jobs:
       - store_artifacts:
           path: coverage
           destination: jest-coverage
-      - run:
-          name: static analysis
-          command: npm run analysis
+      - run: npm run analysis
```

Comment:
Clarity comes from knowing what the actual command is being run, in my opinion.  We could make the script name `static_analysis`, if you like.

Reply:
As mentioned F2F, I kept the npm script name short to avoid typing, but this step is a standalone "extra" thingy that is unlikely to require attention for building docker or whatever.  So this is one I would probably leave alone, or even change the name to "static-analysis with es6-plato" or something.